src/website/views/components/forms/number-input.pug

```diff
@@ -34,58 +32,181 @@ h4 Disabled
 .example.-mb--3
   .-p--3
     .m-formGroup
-      .a-inputWrapper.-compact
+      .a-inputWrapper
         input.a-input(type="number", value="0", disabled)
         button
         button
   :code(lang="html")
-    <div class="a-inputWrapper -compact">
+    <div class="a-inputWrapper">
       <input type="number" class="a-input" value="0" disabled>
       <button></button>
       <button></button>
     </div>
 
-h3 Sizes
+h4 Sizes
 p.-text
-  | Number inputs supports a full spectrum of sizes: <code>-sm</code>, <code>-md</code>,
+  | Number inputs support a full spectrum of sizes: <code>-sm</code>, <code>-md</code>,
   |  <code>-lg</code>, <code>-xl</code>.
   | The default size is <code>-md</code>.
 .example.-mb--3
   .-p--3
     each size in ['sm', 'md', 'lg', 'xl']
       .m-formGroup.-inline.-mr--3
-        .a-inputWrapper.-compact
+        .a-inputWrapper
           input.a-input(type="number", class=`-${size}`, value="0")
           button
           button
   :code(lang='html')
-    <div class="a-inputWrapper -compact">
+    <div class="a-inputWrapper">
       <input type="number" class="a-input -sm" value="0">
       <button></button>
       <button></button>
     </div>
-    <div class="a-inputWrapper -compact">
+    <div class="a-inputWrapper">
       <input type="number" class="a-input -md" value="0">
       <button></button>
       <button></button>
     </div>
-    <div class="a-inputWrapper -compact">
+    <div class="a-inputWrapper">
       <input type="number" class="a-input -lg" value="0">
       <button></button>
       <button></button>
     </div>
-    <div class="a-inputWrapper -compact">
+    <div class="a-inputWrapper">
       <input type="number" class="a-input -xl" value="0">
       <button></button>
       <button></button>
     </div>
 
-h3 JavaScript
-p.-text This component requires JavaScript to increment and decrement the value when the buttons are pressed.
+h3 Expanded version
+
+p.-text
+  | There is available an expanded version of the number input component.
 
-h4 Example
-.-mb--4
-  include:code(lang="js") ../../../../chi/components/input-number/input-number.js
+h4 Base
+.example.-mb--3
+  .-p--3
+    .m-inputNumber
+      input.a-input(type="number", value="0")
+      button.a-btn.-icon
+        .a-btn__content
+          i.a-icon.icon-minus
+      button.a-btn.-icon
```

Comment:
@mattnickles: They were rendering blurry just because they where rendering bigger. There was a bug in buttons with icons, they were being rendered in different size webfonts icons and SVG icons.  The blurry version is the correct 😖. Should I change it?